front/packages/ui/src/home/recommanded.tsx

```diff
@@ -125,7 +127,7 @@ export const ItemDetails = ({
 					})}
 				>
 					<Skeleton {...css({ width: percent(100) })}>
-						{isLoading || <P {...css([{ m: 0 }, "title"])}>{name}</P>}
+						{isLoading || <P {...css([{ m: 0, color: "white" }, "title"])}>{name}</P>}
```

Comment:
you should use theme.colors.white instead of a hardcoded white. Kyoo never uses #ffffff white.